extensions/sql-migration/src/dialog/assessmentResults/sqlDatabasesTree.ts

```diff
@@ -179,8 +179,12 @@ export class SqlDatabaseTree {
 		this._databaseTable.onRowSelected(({ row }) => {
 
 			this._databaseTable.focus();
-			this._activeIssues = this._model._assessmentResults?.databaseAssessments[row].issues;
-			this._selectedIssue = this._model._assessmentResults?.databaseAssessments[row].issues[0];
+			if (this._targetType === MigrationTargetType.SQLMI) {
+				this._activeIssues = this._model._assessmentResults?.databaseAssessments[row].issues;
```

Comment:
will this code continue to work when we add support for SQL DB?  i.e. is there a way to write the code so it doesn't need to be rewritten when we add support for other targets in the future?

Reply:
@kburtram, I agree with that statement. However, at this point it is all over the extension.  We will have make some broader changes to fix this on the extension level. Will add this as work item before going public